CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Main
 
+#### Enhancements
```

Comment:
There's an "Enhancements" section in line 11/20 already. Please add your entry there.